README.md

```diff
@@ -1 +1,44 @@
-# oop-school-library
\ No newline at end of file
+# oop-school-library
+
+## Description
+
+...
```

Comment:
- [ ] Please it will be nice to add a description about your project, as this will make your `Readme file` [professional](https://github.com/microverseinc/readme-template) and will help users to quickly get an idea of what your project is about.

Reply:
Thanks @t-yanick , I already add it.